components/Tabs/Tabs.js

```diff
@@ -1,72 +1,73 @@
-      // this.cards = ;
+      this.cards = new TabCard(this.itemElement);
     }
-    /* <- Delete this comment block when you work on the if statement
+    
 
-     // Map over the newly converted NodeList we just created in our if statement above. Convert each this.cards element into a new instance of the TabCard class. Pass in a card object to the TabCard class. 
-    // this.cards = Array.from(this.cards).map();
+    // Map over the newly converted NodeList we just created in our if statement above. Convert each this.cards element 
+    // into a new instance of the TabCard class. Pass in a card object to the TabCard class. 
+    this.cards = Array.from(this.cards).map(node => new TabCard(node));
 
-    // Add a click event that invokes this.selectTab
-    // this.tabElement.addEventListener();
+    //Add a click event that invokes this.selectTab
+    this.tabElement.addEventListener('click', this.selectTab);
   }
 
   selectTab(){
 
-    // Select all elements with the .tab class on them
-    // const tabs = document.querySelectorAll();
+    //Select all elements with the .tab class on them
+    const tabs = document.querySelectorAll('.tab');
     
-    // Iterate through the NodeList removing the .active-tab class from each element
-    // tabs.forEach()
+    //Iterate through the NodeList removing the .active-tab class from each element
+    Array.from(tabs).forEach(tab => tab.classList.remove('active-tab'));
 
-    // Select all of the elements with the .card class on them
-    // const cards = ;
+    //Select all of the elements with the .card class on them
+    const cards = document.querySelectorAll('.card');
 
-    // Iterate through the NodeList setting the display style each one to 'none'
-    // cards.forEach()
+    //Iterate through the NodeList setting the display style each one to 'none'
+    cards.forEach(cards.style.display = "none");
     
-    // Add a class of ".active-tab" to this.tabElement
-    // this.tabElement;
+    //Add a class of ".active-tab" to this.tabElement
+    this.tabElement.classList.add('active-tab');
   
-    // Notice we are looping through the this.cards array and invoking selectCard() from the TabCard class. Just un-comment the code and study what is happening here.
-    // this.cards.forEach(card => card.selectCard());
+    //Notice we are looping through the this.cards array and invoking selectCard() from the TabCard class. Just un-comment the code and study what is happening here.
+    this.cards.forEach(card => card.selectCard());
   }
 }
 
 class TabCard {
   constructor(cardElement){
     // Assign this.cardElement to the cardElement DOM reference
-    // this.cardElement;
+    this.cardElement = cardElement;
   }
   selectCard(){
     // Update the style of this.cardElement to display = "flex"
-    // this.cardElement;
+    this.cardElement = document.querySelector(this.cardElement).style.display = "flex";
```

Comment:
You wouldn't need to us a querySelector to grab this.cardElement... this task could be accomplished like this:
selectCard(){
    this.cardElement.style.display = 'flex';
  }